### DISCUSSION TITLE
VERY SLOWLY to plot..HELP!

### DISCUSSION THREAD

**Discussion by leochang36** [Chia-Network/chia-blockchain#2809] [Q&A]
CPU：i3-10100
RAM：64GB
Disk：NVME 1T+16THDD（Through NFS to my NAS)
Environment：
windows 10 in ESXi7.0,Plot a file maybe need 50 HOURS....
I tested my SSD Speed and RAM speed all normally....
So,I don't know what problem with this plotting....
Thanks!

**Comment by dorkmo** [comment]
it might be your NAS. Can you connect the drives directly to your plotter?

**Comment by Ero-Cat** [comment]
me too.

**Comment by Keriwaza** [comment]
Best to plot to local drive then transfer to NAS. Network speed may hurt you using the NAS as final destination

**Comment by max918** [comment]
plotting in VM will let you more slowly

**Reply by Ero-Cat** [reply]
在esxi 里具体效率会降低多少呢？

https://zhuanlan.zhihu.com/p/271901267

**Comment by actuaryck** [comment]
I have exactly the same issue. I am running on a Ryzen 5 3600 16GB RAM machine and plotting on a RAID-0 storage composed of three 7200rpm HDD. My first plot took 61 hours to finish. And 67 hours has passed, my second and third plot (I ran them in parallel) are still on 70% and 37%.

I see people finishing several plots in a day and I wonder what has gone wrong with my system.

**Reply by actuaryck** [reply]
I was running the above RAID-0 using a LSI 9260-8i array controller. After I switched to the onboard SATA controller, the plotting speed seems to be normal. I can now do around four plots per day on a 4-disk RAID-0 or a single NVMe SSD drive (still not very fast).

**Comment by psun888** [comment]
Without pool, it's truly wasting time and energy with such small setup. We have many servers doing plotting and more than 300TB farming, for almost one week without any chia farmed. There are so many bugs and problems in Chia network, new farmers should be very cautious with significant investment and energy.

**Comment by Ero-Cat** [comment]
Intel 660P 1T M.2 2280 PCIe NVMe 

ESXi6.7 Windows10

![image](https://user-images.githubusercontent.com/11071219/116633435-09cede80-a98c-11eb-85ae-b5c76023f40c.png)
PC Windows10

![image](https://user-images.githubusercontent.com/11071219/116639360-f5dea900-a99a-11eb-87fd-98ae3a5d4262.png)

**Reply by Ero-Cat** [reply]
sad ,经过如上测试 ，我要把ESXi删除重做系统

**Reply by Ero-Cat** [reply]
https://zhuanlan.zhihu.com/p/271901267

**Comment by nomadengineer** [comment]
CPU also plays a significant role in the first phase. Having a more powerful CPU allows for faster plotting.
I have so far tried on 2 types intel servers and an AMD server. The AMDs are at least twice as slow as intel!
